src/components/Card/__stories__/Card.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
+import {Card, CardProps} from '../Card';
+import {CardShowcase} from './CardShowcase';
+
+import './Card.stories.scss';
+
+export default {
+    title: 'Components/Card',
+    customPath: 'Components/Card',
```

Comment:
paths are the same, this line can be deleted

Reply:
ec48a2d